doc/manual/queryformat

```diff
@@ -153,10 +154,10 @@ braces.
 
 \section queryformat_example Example: Viewing the Verify Flags
 
-The following example query is run against dev because I know %verify
-is used there.
+The following example query is run against nss because %verify is used
+there.
```

Comment:
I agree the old sentence is pretty bad, but this is not much of an improvement. "nss" doesn't go by that name everywhere, the packaging details such as the name and use of %verify are highly distro specific. These days, "systemd" might well be the most ubiquitous name there is, but that doesn't make %verify flags any less distro specific.